config/settings_schema.json

```
@@ -133,6 +133,63 @@
       }
     ]
   },
+  {
+    "name": "Grid",
+    "settings": [
+      {
+        "type": "paragraph",
+        "content": "Affects collection, search and blog pages. Also affects collage, collection, collection list, multicolumn and blog sections"
```

```suggestion
        "content": "Affects collection, search and blog pages. Also affects collage, collection, collection list, multicolumn and blog sections."
```

Just adding a period at the end. 
The content will be revise later but I like giving more clarity of where this applies when it is complex. I like that we won't let the user think/question where the setting applies.